Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalTBModule.cc

```diff
@@ -11,7 +11,7 @@
 #ifdef EDM_ML_DEBUG
 #include <unordered_set>
 #endif
-using namespace geant_units::operators;
+using namespace cms_units::operators;
```

Comment:
@ianna is this change addressing the difference in units of measurements assumed within DD4hep and out Geant-oriented geometry description? ```piRadians``` stays the same